packages/x-date-pickers-pro/src/internal/hooks/useStaticRangePicker/useStaticRangePicker.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
```

Comment:
Nitpick: Maybe it would be worth trying to avoid `@mui/material` imports for new code whenever possible if we have plans to support Joy or other UI in the future?
In this case the following should work as well 🤔 
```suggestion
import { styled } from '@mui/system';
```

Reply:
We should check with the core team, but for me both are not equivalent.
They don't have the same default `theme` and `shouldForwardProp`.

That would be an interesting topic to discuss when we will want to stop relying on `@mui/material`.

Reply:
Mhh, I didn't check all the code. Sorry for confusing on this one.
We will have to bring this topic to them someday. 👍